xla/stream_executor/dnn.cc

```diff
@@ -246,10 +247,49 @@ DnnSupport::FusedMHAScaleBiasSoftmaxRunnerFromDesc(
     const dnn::TensorDescriptor& output_descriptor,
     const dnn::TensorDescriptor& bias_descriptor, double scale,
     std::optional<double> dropout_rate, std::optional<int64_t> seed) {
-  return tsl::errors::Unimplemented(
+  return absl::UnimplementedError(
       "FusedMHAScaleBiasSoftmaxRunnerFromDesc not implemented.");
 }
 
+tsl::StatusOr<std::unique_ptr<const dnn::FusedMHASoftmaxBackwardRunner>>
+DnnSupport::FusedMHASoftmaxBackwardRunnerFromDesc(
+    Stream* stream, const dnn::AlgorithmDesc& algorithm_desc,
+    dnn::FusedMHAKind kind,
+    const MatmulTensorDescriptor& bmm1_grad_gemm1_rhs_descriptor,
+    const MatmulTensorDescriptor& bmm1_grad_gemm2_rhs_descriptor,
+    const MatmulTensorDescriptor& bmm2_grad_gemm1_lhs_descriptor,
+    const MatmulTensorDescriptor& bmm2_grad_gemm2_rhs_descriptor,
+    const MatmulTensorDescriptor& d_output_descriptor,
+    const TensorDescriptor& d_bmm1_lhs_descriptor,
+    const TensorDescriptor& d_bmm1_rhs_descriptor,
+    const TensorDescriptor& d_bmm2_rhs_descriptor,
+    const TensorDescriptor& d_S_descriptor,
```

Comment:
Optional: the style guide complains about this, it wants this to be `d_s_descriptor`, but it occurs to me the S might be getting at some standard notation I'm not aware of. Whatever you think is best here works for me

Reply:
changed all capital to lower case for this variable